packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -114,6 +114,19 @@ const ChildrenContainer = styled("div", {
   },
 }));
 
+const StyledMuiCard = styled(MuiCard, {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{ odysseyDesignTokens: DesignTokens }>(({ odysseyDesignTokens }) => ({
+  boxShadow: "none",
+  "&::after": {
+    opacity: 0,
+    "box-shadow": odysseyDesignTokens.DepthMedium,
```

Comment:
We'll need to transition the opacity here as well

Reply:
there's a transition already in the overrides for `MuiCard`. do we need to override that?

Reply:
Hmm, I see that. I don't think we really want to transition "all". I'd assume that was put in place for `box-shadow` only. But, we'd have to ask. @jordankoschei-okta Any additional context here? 

Ideally the `box-shadow` transition goes away and we are only transitioning `opacity`. That would involve moving the `MuiCard` to this same pseudo element solution though